tests/Command/RunCommandTest.php

```diff
@@ -1101,7 +1102,8 @@ public function testExecuteStateTransitionError(): void
             $objectEncryptor,
             '123',
             (string) getenv('TEST_STORAGE_API_TOKEN'),
-            []
+            [],
+            new UuidGenerator()
```

Comment:
proč tady není rovnou (string) Uuid::v4(); ?

Reply:
Mohlo by to byt i tak, pak bych to nemusel mockovat v tech testech vlastne. Ja jsem to chtel udelat pres service container, ale slo by to i tak jak pises.

Reply:
Joo nee, musi to byt takhle. V testech by to slo udelat, protoze tam muzes ten string runnerId dat do konstruktoru, ale v service containeru nemuzes hezky udelat factory na argument.

Reply:
imho to vůbec nemusí být v konstruktoru a ni v DI, ani ta classa tu nemusí být, prostě se jen tady https://github.com/keboola/job-runner/pull/171/files#diff-f5b5ee78333f904a0e7b00e1c283110a5c85ad7533654c2906453e9c9d4c16a7R43 podívat, že v jobPatchData je neprázdný runnerId

Reply:
OK, tak som to zas vratil viac-menej ako to bolo, akurat som pridal test aby to nebolo empty